microsoft-edge/webview2/concepts/multi-profile-support.md

```diff
@@ -6,84 +6,152 @@ ms.author: msedgedevrel
 
-This method creates a WebView2 control and associates it with the profile you specified. If the specified profile does not exist, a new one will be created.
+##### [WinRT/C#](#tab/winrtcsharp)
 
+<!-- Ref topic breakout: one webpage per type; very long webpage covers all methods of the type including overloads.  2nd overload's url adds -1 at end -->
+<!-- preserve sequence per Ref webpage.  for clarity + brevity, list param names, lowercased -->
+* [CoreWebView2Environment.CreateCoreWebView2CompositionControllerAsync(parentWindow) Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createcorewebview2compositioncontrollerasync)
+* [CoreWebView2Environment.CreateCoreWebView2CompositionControllerAsync(parentWindow, options) Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createcorewebview2compositioncontrollerasync-1)
+* [CoreWebView2Environment.CreateCoreWebView2ControllerAsync(parentWindow, options) Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createcorewebview2controllerasync)
+* [CoreWebView2Environment.CreateCoreWebView2ControllerAsync(parentWindow) Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createcorewebview2controllerasync-1)
 
-<!-- ------------------------------ -->
+##### [Win32/C++](#tab/win32cpp)
 
-# [C#](#tab/csharp)
+<!-- Ref topic breakout: small dedicated iface.  link to iface to bring up overview, and link to methods to show method names -->
+* **[ICoreWebView2Environment10 interface](/microsoft-edge/webview2/reference/win32/icorewebview2environment10)**
+   * [ICoreWebView2Environment10::CreateCoreWebView2ControllerWithOptions method](/microsoft-edge/webview2/reference/win32/icorewebview2environment10#createcorewebview2controllerwithoptions)
+   * [ICoreWebView2Environment10::CreateCoreWebView2CompositionControllerWithOptions method](/microsoft-edge/webview2/reference/win32/icorewebview2environment10#createcorewebview2compositioncontrollerwithoptions)
 
-TODO
+---
 
 
-<!-- ------------------------------ -->
+<!-- ====================================================================== -->
+## Example of specifying the profile when creating a WebView2
 
-# [C++](#tab/cpp)
 
-* **[ICoreWebView2Environment10](/microsoft-edge/webview2/reference/win32/icorewebview2environment10)**
-   * [CreateCoreWebView2ControllerWithOptions](/microsoft-edge/webview2/reference/win32/icorewebview2environment10#createcorewebview2controllerwithoptions)
-   * [CreateCoreWebView2CompositionControllerWithOptions](/microsoft-edge/webview2/reference/win32/icorewebview2environment10#createcorewebview2compositioncontrollerwithoptions)
+##### [.NET/C#](#tab/dotnetcsharp)
 
----
+<!-- TODO: replace code listing? -->
```

Comment:
@leonhsl - Replace the below C# code that's copied from spec?
WinRT section has same code.